lib/mplcairo/qt.py

```diff
@@ -1,7 +1,12 @@
 import ctypes
 
 # This does support QT_API=pyqt6 on Matplotlib versions that can handle it.
-from matplotlib.backends.backend_qt5 import _BackendQT5, FigureCanvasQT
+try:
+    from matplotlib.backends.backend_qt import _BackendQT, FigureCanvasQT
+except ImportError:
+    from matplotlib.backends.backend_qt5 import (
+        _BackendQT5 as _BackendQT, FigureCanvasQT
+    )
```

Comment:
move parenthesis to previous line (if the in-parenthesis contents are not one-item-per-line, the "minimize diff" argument does not apply).